python-package/lightgbm/compat.py

```diff
@@ -14,14 +14,6 @@
     from sklearn.utils.multiclass import check_classification_targets
     from sklearn.utils.validation import assert_all_finite, check_array, check_X_y
 
-    # sklearn.utils Tags types can be imported unconditionally once
-    # lightgbm's minimum scikit-learn version is 1.6 or higher
-    try:
-        from sklearn.utils import ClassifierTags as _sklearn_ClassifierTags
-        from sklearn.utils import RegressorTags as _sklearn_RegressorTags
-    except ImportError:
-        _sklearn_ClassifierTags = None
-        _sklearn_RegressorTags = None
```

Comment:
Please remove these lines too:
https://github.com/microsoft/LightGBM/blob/ea04c66c86e31ebf68ec151d75c14fbdfb6ea681/python-package/lightgbm/compat.py#L151-L152

Reply:
blegh sorry for missing those! Removed in https://github.com/microsoft/LightGBM/pull/6735/commits/66033970719da915db655fb8c7dcc1bb2f7e636f